src/client/src/services/model/currencyPairPosition.js

```diff
@@ -1,10 +1,12 @@
 export default class CurrencyPairPosition {
   _symbol:string;
+  _ccy:string;
   _basePnl:number;
   _baseTradedAmount:number;
 
   constructor(symbol:string, basePnl:number, baseTradedAmount:number) {
     this._symbol = symbol;
+    this._ccy = symbol.length === 6 ? symbol.substr(0, 3) : '';
```

Comment:
What is currency?  The base or terms?  I think the naming is slightly off.
Is there ever a case where symbol is not a fully fledged currency pair and is something else?  If it isn't, then this sounds like an error.